pkg/manifests/types.go

```diff
@@ -19,377 +19,398 @@ import (
-	// a sample. If a label value is longer than this number post metric-relabeling, the entire scrape will
-	// be treated as failed. 0 means no limit.
-	// default: 0
+	// Specifies a per-scrape limit on the length of a label value for
+	// a sample. If a label value exceeds this limit after 
+	// metric relabeling, the entire scrape is treated as failed.
+	// The default value is `0`, which means that no limit is set.
 	EnforcedLabelValueLengthLimit *uint64 `json:"enforcedLabelValueLengthLimit,omitempty"`
-	// EnforcedSampleLimit defines a global limit on the number of scraped samples that will be accepted.
-	// This overrides any SampleLimit set per ServiceMonitor or/and PodMonitor. It is meant to be
-	// used by admins to enforce the SampleLimit to keep the overall number of samples/series under the
-	// desired limit. Note that if SampleLimit is lower that value will be taken instead.
-	// default: 0
+	// Specifies a global limit on the number of scraped samples that will be accepted.
+	// This setting overrides any `SampleLimit` value set per `ServiceMonitor` or `PodMonitor`. 
+	// Administrators can use this setting to enforce a `SampleLimit` value and 
+	// keep the overall number of samples or series under the set limit. 
+	// However, if the `SampleLimit` value is lower than this limit,
+	// the `SampleLimit` value is used instead.
+	// The default value is `0`.
 	EnforcedSampleLimit *uint64 `json:"enforcedSampleLimit,omitempty"`
-	// EnforcedTargetLimit defines a global limit on the number of scraped targets. This overrides
-	// any TargetLimit set per ServiceMonitor or/and PodMonitor. It is meant to be used by admins to
-	// enforce the TargetLimit to keep the overall number of targets under the desired limit. Note
-	// that if TargetLimit is lower, that value will be taken instead, except if either value is
-	// zero, in which case the non-zero value will be used. If both values are zero, no limit is
-	// enforced.
-	// default: 0
+	// Specifies a global limit on the number of scraped targets. This value overrides
+	// any `TargetLimit` value set per `ServiceMonitor` or `PodMonitor`. 
+	// Administrators can use this setting to enforce a `TargetLimit` value and
+	// keep the overall number of targets under the set limit. Note
+	// However, if the `TargetLimit` value is lower than this limit, 
+	// the `TargetLimit` value is used instead. If one of the values is
+	// `0`, the non-zero value is used. If both values are 
+	// `0`, no limit is set.
+	// The default value is `0`.
 	EnforcedTargetLimit *uint64 `json:"enforcedTargetLimit,omitempty"`
-	// ExternalLabels defines labels to be added to any time series or alerts when communicating
-	// with external systems (federation, remote storage, Alertmanager).
-	// default: nil
+	// Defines labels to be added to any time series or alert when 
+	// communicating with external systems such as federation, remote storage, and Alertmanager.
+	// The default value is `nil`.
```

Comment:
same as for the PrometheusK8sConfig struct

Reply:
fixed